.github/workflows/update-tables.yml

```diff
@@ -0,0 +1,48 @@
+name: Update Syscalls Tables
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '0 1 * * 1' # Each monday
```

Comment:
I prefer Tuesday as safer day. Nevermind which timezone Linus Torvalds will be RC will be already done.